<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Animation</class>
 <widget class="QWidget" name="Animation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>981</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,1,1,0">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_28">
         <item>
          <widget class="QRadioButton" name="rbtnSnapTimeMode">
           <property name="text">
            <string>  原始时间</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbtnInterpolateTimeMode">
           <property name="text">
            <string> 插值时间</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_29" stretch="1,1,1,1">
         <item>
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>动画步长</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBoxAnimationStride">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>关键帧数量</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditKeyframeNum"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_30" stretch="0,2">
         <item>
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>开始时间</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditStartTime"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_31" stretch="1,2">
         <item>
          <widget class="QLabel" name="label_21">
           <property name="text">
            <string>结束时间</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditEndTime"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_32" stretch="1">
         <item>
          <widget class="QPushButton" name="btnApplyAnimationOperation">
           <property name="text">
            <string>应用操作</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_20" stretch="1,0,0,0,0,0,0,0,1">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnFirstFrame">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;最初帧&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="whatsThis">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;最初帧&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../iGameQtMainWindow.qrc">
             <normaloff>:/Ticon/Icons/VcrFirst.svg</normaloff>:/Ticon/Icons/VcrFirst.svg</iconset>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnPreviousFrame">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;上一帧&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="whatsThis">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;上一帧&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../iGameQtMainWindow.qrc">
             <normaloff>:/Ticon/Icons/VcrBack.svg</normaloff>:/Ticon/Icons/VcrBack.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnReverseOrPause">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;倒序播放&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="whatsThis">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;倒序播放&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../iGameQtMainWindow.qrc">
             <normaloff>:/Ticon/Icons/VcrReverse.svg</normaloff>:/Ticon/Icons/VcrReverse.svg</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnPlayOrPause">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;顺序播放&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="whatsThis">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;顺序播放&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../iGameQtMainWindow.qrc">
             <normaloff>:/Ticon/Icons/VcrPlay.svg</normaloff>:/Ticon/Icons/VcrPlay.svg</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnNextFrame">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;下一帧&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="whatsThis">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;下一帧&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../iGameQtMainWindow.qrc">
             <normaloff>:/Ticon/Icons/VcrForward.svg</normaloff>:/Ticon/Icons/VcrForward.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnLastFrame">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;最后帧&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="whatsThis">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;最后帧&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../iGameQtMainWindow.qrc">
             <normaloff>:/Ticon/Icons/VcrLast.svg</normaloff>:/Ticon/Icons/VcrLast.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnLoop">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;开始循环&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../iGameQtMainWindow.qrc">
             <normaloff>:/Ticon/Icons/VcrDisabledLoop.svg</normaloff>:/Ticon/Icons/VcrDisabledLoop.svg</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSlider" name="SliderAnimationTrack">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="igQtAnimationTreeWidget_snap" name="treeWidget_snap">
         <column>
          <property name="text">
           <string notr="true">文件原始时间</string>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>值</string>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="igQtAnimationTreeWidget_interpolate" name="treeWidget_interpolate">
         <column>
          <property name="text">
           <string notr="true">插值时间序列</string>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>值</string>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>igQtAnimationTreeWidget_snap</class>
   <extends>QTreeWidget</extends>
   <header>IQComponents/igqtanimationtreewidget_snap.h</header>
  </customwidget>
  <customwidget>
   <class>igQtAnimationTreeWidget_interpolate</class>
   <extends>QTreeWidget</extends>
   <header>IQComponents/igqtanimationtreewidget_interpolate.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../iGameQtMainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
